databricks_cli/pipelines/cli.py

```diff
@@ -49,28 +50,147 @@
 @click.option('--allow-duplicate-names', is_flag=True,
-              help="If true, skips duplicate name checking while deploying the pipeline.")
+              help="If true, skips duplicate name checking while creating the pipeline.")
+@debug_option
+@profile_option
+@pipelines_exception_eater
+@provide_api_client
+def create_cli(api_client, settings_arg, settings, allow_duplicate_names):
+    """
+    Creates a pipeline according to the pipeline settings. The pipeline settings are a
+    JSON document that defines a Delta Live Tables pipeline on Databricks. If the pipeline
+    creation is successful, logs the URL and the ID of the new pipeline to STDOUT.
+
+    If a pipeline with the same name already exists, the pipeline will not be created.
+    This check can be disabled by adding the --allow-duplicate-names option.
+
+    If the pipeline settings contain an "id" field, this command will fail.
+
+    Usage:
+
+    databricks pipelines create example.json
+
+    OR
+
+    databricks pipelines create --settings example.json
+    """
+    if bool(settings_arg) == bool(settings):
+        raise ValueError('Settings should be provided either as an argument ' +
+                         '(Eg: databricks pipelines create example.json) or as ' +
+                         'an option (Eg: databricks pipelines create --settings example.json).')
+
+    src = settings_arg if bool(settings_arg) else settings
+    settings_obj = _read_settings(src)
+    settings_dir = os.path.dirname(src)
+
+    if 'id' in settings_obj:
+        raise ValueError("Pipeline settings shouldn't contain \"id\" "
+                         "when creating a pipeline.")
+
+    try:
+        response = PipelinesApi(api_client).create(
+            settings_obj, settings_dir, allow_duplicate_names)
+    except requests.exceptions.HTTPError as e:
+        _handle_duplicate_name_exception(settings_obj, e, is_create_pipeline=True)
+
+    new_pipeline_id = response['pipeline_id']
+    click.echo("Successfully created pipeline: {} with ID: {}.".format(
+        _get_pipeline_url(api_client, new_pipeline_id), new_pipeline_id))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Edits a pipeline with the given pipeline settings.')
+@click.argument('settings_arg', default=None, required=False)
+@click.option('--settings', default=None, type=PipelineSettingClickType(),
+              help=PipelineSettingClickType.help)
+@click.option('--pipeline-id', default=None, type=PipelineIdClickType(),
+              help=PipelineIdClickType.help)
+@click.option('--allow-duplicate-names', is_flag=True,
+              help="Skip duplicate name check while editing pipeline.")
+@debug_option
+@profile_option
+@pipelines_exception_eater
+@provide_api_client
+def edit_cli(api_client, settings_arg, settings, pipeline_id, allow_duplicate_names):
+    """
+    Edits a pipeline according to the pipeline settings. The pipeline settings are a
```

Comment:
"Edits a pipeline. The new pipeline settings are defined in a JSON document. The JSON settings document can be passed directly to the command or stored in a file. To use a file containing the pipeline settings, use the --settings argument to pass the file path to the command. "

Reply:
It looks like this also outputs the URL and ID on success. Should we add that?